src/qt/bitcoin.cpp

```diff
@@ -466,6 +468,10 @@ int GuiMain(int argc, char* argv[])
     QApplication::setOrganizationName(QAPP_ORG_NAME);
     QApplication::setOrganizationDomain(QAPP_ORG_DOMAIN);
     QApplication::setApplicationName(QAPP_APP_NAME_DEFAULT);
+    if (gArgs.GetArg("-guisettings", "") != "") {
+        QSettings::setDefaultFormat(QSettings::IniFormat);
```

Comment:
[`QSettings::IniFormat`](https://doc.qt.io/qt-5/qsettings.html#Format-enum):
> ... Note that type information is not preserved when reading settings from INI files; all values will be returned as `QString`.

Could this cause any issue?

Reply:
At the moment we already use the IniFormat on all platforms as far as I know it.

Reply:
Good question. On UNIX it is always in IniFormat. On macOS and Windows it is in the registry (even if that's not the term on macOS). What would be the point in selecting a custom path to the registry? I don't know if this is even possible.

Reply:
I think this as resolved as it is not possible (at least not by ease) to store registry stuff outside of the registry. `IniFormat` is the only cross platform settings format supported by Qt and it is only used with if the parameter is set.